### ISSUE TITLE
sandpass entry with no title

### ISSUE THREAD

**Issue by sakibmas** [zombiezen/sandpass#30]
There is only a dot type symbol for sandpass entry which has no title. So unable to edit or delete it.

**Comment by zombiezen** [comment]
Yeah, this should probably be a required field in client-side and should also have a placeholder in the UI. Happy to accept a PR.

As a workaround, you can download a backup of your grain, open up the database in KeePass, and then restore a new grain with the fixed database.
